webapp/src/DataFixtures/DefaultData/UserFixture.php

```diff
@@ -0,0 +1,79 @@
+<?php declare(strict_types=1);
+
+namespace App\DataFixtures\DefaultData;
+
+use App\Entity\User;
+use App\Service\DOMJudgeService;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Persistence\ObjectManager;
+
+class UserFixture extends AbstractDefaultDataFixture implements DependentFixtureInterface
+{
+    /**
+     * @var DOMJudgeService
+     */
+    protected $dj;
+
+    public function __construct(DOMJudgeService $dj)
+    {
+        $this->dj = $dj;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function load(ObjectManager $manager)
+    {
+        if (!($adminuser = $manager->getRepository(User::class)->findOneBy(['username' => 'admin']))) {
+            $adminPasswordFile      = sprintf(
+                '%s/%s',
+                $this->dj->getDomjudgeEtcDir(),
+                'initial_admin_password.secret'
+            );
+            $encryptedAdminPassword = password_hash(
```

Comment:
Should we not use the same function we use in other places to create passwords?

Reply:
We should.